<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0.5, y2:0.5, stop:0 rgba(50, 50, 50, 200), stop:1 rgba(190, 255, 255, 255));</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica Neue</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 24pt &quot;Helvetica Neue&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Bienvenido a Alzhense</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="texto">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>441</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 11pt &quot;Helvetica Neue&quot;;</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>Alzhense es una plataforma que cuenta con un estimador de tipo random forest para predecir el diagnóstico positivo o negativo de enfermedad de Alzheimer, en base a los datos del paciente y las imágenes de su resonancia magnética cerebral.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignJustify|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Avanzar">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;
font-weight:bold</string>
   </property>
   <property name="text">
    <string>Siguiente</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EntrenarRed">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>13</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Entrenar el clasificador</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MostrarMatriz">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50 , 50);
border-radius: 15px;
font: 75 13pt &quot;Helvetica&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Mostrar resultados</string>
   </property>
  </widget>
  <widget class="QLabel" name="Imagen">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>351</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Entrenando">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>9</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 9pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PreLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ExLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;
font-weight:bold;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="SenLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="EsLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>350</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Exactitud">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Precision">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>290</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Sensibilidad">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>320</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Especificidad">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>350</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 75 12pt &quot;Helvetica&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Porc1">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>260</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Porc2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>290</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Porc3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>320</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Porc4">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>350</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 12pt &quot;Helvetica&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ResultadosLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Myriad Pro</family>
     <pointsize>13</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
font: 75 13pt &quot;Myriad Pro&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
